pysph/base/tests/test_opencl.py

```diff
@@ -4,8 +4,55 @@
 
 pytest.importorskip("pysph.base.opencl")
 
+import pyopencl as cl
+
 from pysph.base.utils import get_particle_array  # noqa: E402
-from pysph.base.opencl import DeviceHelper  # noqa: E402
+from pysph.base.opencl import DeviceHelper, DeviceArray, \
+        get_queue  # noqa: E402
+
+
+class TestDeviceArray(TestCase):
+    def test_reserve(self):
+        # Given
+        self.dev_array = DeviceArray(np.int32, n=16)
```

Comment:
Well, you don't even need `self.dev_array` as you can just do `dev_array = ...` and the code becomes even simpler!